src/transformers/models/clip/configuration_clip.py

```diff
@@ -118,6 +120,94 @@ def __init__(
         self.initializer_factor = initializer_factor
         self.attention_dropout = attention_dropout
 
+    @classmethod
+    def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike], **kwargs) -> "PretrainedConfig":
+        r"""
```

Comment:
The docstring will be automatically inherited from the superclass, so you should remove it here.